File: example2.py
from pyspark import SparkConf, SparkContext
con=SparkConf()
sc=SparkContext(conf=con)
dic1=[1,2,3]
dic2=[5]
rdd=sc.parallelize(dic1)
rdd2=sc.parallelize(dic2)
rdd3=rdd.union(rdd2)
def toWords(x):
	abc={1:"One",2:"Two",3:"Three",4:"Four"}
	return(abc[x])
x2=rdd.map(toWords)
thisData=x2.collect()
print(thisData)

File: example2.scala
import org.apache.spark.SparkContext
import org.apache.spark.SparkConf

object thisthang {
  def main(args: Array[String]){
var con=new SparkConf()
var sc=new SparkContext(conf=con)

var rdd=sc.parallelize(array(1,2,3,4,5))
var theList=rdd.collect()

for(a <- theList){
println(a)
}
}}
